samples/snippets/src/main/java/com/example/spanner/HubbleTransactionsCodeLab.java

```diff
@@ -14,18 +14,21 @@
 
+  private static final String SUBJECT = "Random subject for the email for testing purpose %s";
+
   public void createMessages(DatabaseAdminClient dbAdminClient, DatabaseId id) {
     createDatabase(
         dbAdminClient,
         id,
-        Arrays.asList(
+        Collections.singletonList(
             "CREATE TABLE Message("
                 + " msg_id STRING(MAX) NOT NULL,"
                 + " body STRING(MAX),"
                 + ") PRIMARY KEY (msg_id)"));
+    System.out.println("Finished execution of createMessages()");
```

Comment:
Is this for debugging? Do we need these prints?

Reply:
I just thought it will give better visibility to user, because when I was running I could not figure out if the method is finished or not. Like sometimes I see if while loop is running for 5 min and finished. I still could not figure out.

Reply:
These functions are generally called by themselves, and this line will print and then the function will return. In this case, the print is just noise - the function returns and it will have obviously finished. Let's remove these where possible, please.